tests/test.py

```diff
@@ -1012,6 +1031,10 @@ def get_connector_tls_params(self):
             'tls_ca': TEST_TLS_CA,
             'tls_key': TEST_TLS_CLIENT_KEY,
             'tls_cert': TEST_TLS_CLIENT_CERT,
+            'tls_ocsp_url': 'http://127.0.0.1:{}'.format(self.OCSP_SERVER_PORT),
```

Comment:
better to use `localhost` instead 127.0.0.1 if will be used TLS in a future and certificates will use localhost in SAN

Reply:
Done